resources/skins.citizen.styles/components/Header.less

```diff
@@ -157,18 +157,47 @@
 
 @media ( min-width: @min-width-breakpoint-desktop ) {
 	.citizen-header {
-		--header-direction: column;
 		top: 0;
 		right: unset;
 		left: 0;
 		border-top: 0;
-		border-right: var( --border-base );
-
-		&__logo {
+		
+		.citizen-header-position-left &__logo,
+		.citizen-header-position-right &__logo {
 			padding: 0 0 var( --space-xs ) 0;
 			margin: var( --space-xxs ) 0;
 			border-right: 0;
-			border-bottom: var( --border-subtle );
+			border-bottom: var( --border-width-base ) solid var( --border-color-base );
```

Comment:
Maybe it makes more sense to set the `border-color` and `border-style` in the logo element, and then change the width on each one.